validphys2/src/validphys/coredata.py

```diff
@@ -393,28 +393,37 @@ def export_data(self, buffer):
         ret = {"data_central": self.central_values.tolist()}
         yaml.safe_dump(ret, buffer)
 
+    def sort_definitions_by_treatment(self, definitions, orderkey):
+        sorted_definitions={}
+        for orderkey in orderkey:
```

Comment:
```suggestion
        for orderkey in orderkey:
```
they are the same name?